### ISSUE TITLE
Zack - Add Load Balancer to your quotes API

### ISSUE THREAD

**Issue by joshuarose** [pgdevelopers/apprenticeship#30]
![Image](https://user-images.githubusercontent.com/1481586/190998112-a9001f15-1e58-42e5-a0f2-fff997ca3795.png)

- Add HTTPs Load balancer in front of your Cloud Run service (NAME-service), you will need to create a Serverless NEG (Network Endpoint Group

-  Change your cloud run service to only except traffic from "Internal or load balancer"

- Test your API with Postman through the Load balancer

**Comment by joshuarose** [comment]
Hey Zack! Your API works great at http://34.149.8.254/quote can we update it to be http://34.149.8.254/quotes?

**Comment by zraney** [comment]
Yes, I attempted to do that yesterday and will keep pushing until I get it!
